### PR TITLE
all: fix go vet complaints

### PR DESCRIPTION
**Pull request by fhs**

### CODE REVIEW

File: dbw/types.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ func (v Vector) Slice(l, r int) Vector {
 		panic("blas: index out of range")
 	}
 	if r > l {
-		panic(fmt.Sprintf("blas: invalid slice index:", r, ">", l))
+		panic(fmt.Sprint("blas: invalid slice index: ", r, " > ", l))
```
Comment (kortschak) [inline dbw/types.go]: Use `fmt.Sprintf("blas: invalid slice index: %d > %d", r, l)`.

File: zbw/types.go [hunk 2]
```diff
@@ -152,7 +152,7 @@ func (v Vector) Slice(l, r int) Vector {
 		panic("blas: index out of range")
 	}
 	if r > l {
-		panic(fmt.Sprintf("blas: invalid slice index:", r, ">", l))
+		panic(fmt.Sprint("blas: invalid slice index: ", r, " > ", l))
```
Comment (kortschak) [inline zbw/types.go]: As above, use fmt.Sprintf.

**Comment by fhs** [comment]
PTAL. I was trying to keep the changes minimal. I think the nested-ifs for checking blas.Transpose should really be written using a switch and probably factored out into a function. I could also make this change if acceptable within this PR.

**Comment by kortschak** [comment]
A lot of this will be rewritten later - there are discusions that need to be had.

LGTM

**Pull request merged**